config/samples/mysql_v1alpha1_cluster.yaml

```diff
@@ -77,4 +79,4 @@ spec:
     accessModes:
     - ReadWriteOnce
     #storageClass: ""
-    size: 10Gi
+    size: 30Gi
```

Comment:
The default is 10Gi, dont modify

Reply:
How about to Change to 20Gi? KDE is limited to 20Gi

Reply:
ok,change to 20Gi